code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -2043,8 +2043,11 @@ datum
 
 
 				else if (method == TOUCH)
-					if(iscarbon(M))
-						if(!M.wear_mask)
+					if(iscarbon(M) || ismobcritter(M))
+						var/mob/living/carbon/human/H = M
```

Comment:
Mob critters aren't `mob/living/carbon/human`.

Reply:
Aight, added a var for critters.